src/main/resources/com/google/api/codegen/csharp/lro_sample.snip

```diff
@@ -0,0 +1,60 @@
+@extends "csharp/common.snip"
```

Comment:
nit: in this and other files, if the order of `@extends` does not matter, I suggest alphabetical order